docs/tutorials/javascript/index.md

```diff
@@ -16,9 +14,9 @@ Typically, the example functions involve four steps.
 - Get the state of accounts and tokens on the XRP Ledger using requests.
 - Disconnect from the XRP Ledger.
 
-Each lesson builds the Token Test Harness one section at a time, with complete JavaScript and HTML code samples and a code walkthrough. You can download the source code, manipulate it in a text editor, and run it in your favorite browser. 
+Each lesson features complete JavaScript and HTML code samples and a code walkthrough. You can download the source code, manipulate it in a text editor, and run it in your favorite browser. 
 
-Much of this is “brute force” code that sacrifices conciseness for readability. Each example builds on the previous examples, adding a new JavaScript file and the supporting HTML UI. 
+Much of this is “brute force” code that sacrifices conciseness for readability. Most examples incorporate functions from earlier examples so that any new code is specific to the topic of that tutorial  
```

Comment:
```suggestion
Much of this is “brute force” code that sacrifices conciseness for readability. Most examples incorporate functions from earlier examples so that any new code is specific to the topic of that tutorial.
```